src/nvim/os/env.c

```diff
@@ -239,29 +239,29 @@ void expand_env(char_u *src, char_u *dst, int dstlen)
 /// "~/" is also expanded, using $HOME. For Unix "~user/" is expanded.
 /// Skips over "\ ", "\~" and "\$" (not for Win32 though).
 /// If anything fails no expansion is done and dst equals src.
-/// startstr recognize the start of a new name, for '~' expansion.
+/// prefix recognize the start of a new name, for '~' expansion.
```

Comment:
I just renamed the variable and kept the old confusing comment.

This argument is used when you want to expand `~` after some prefix (e.g. `myprefix:~/felipe`). I think we should just drop this comment line.